docs/quickstart/README.md

```diff
@@ -5,19 +5,19 @@ In this tutorial, we will show how to configure Tornjak with a SPIRE deployment
 This tutorial will get you up and running with a local deployment of SPIRE and Tornjak in three simple steps: setting up the deployment files, deployment, and connecting to Tornjak. 
 
 Contents
-1. [Requirements](#step-0-requirements)
-2. [Setup of Deployment files](#step-1-setup-quickstart-tutorial)
-3. [Deployment of SPIRE and co-located Tornjak](#step-2-deployment)
-4. [Connecting to Tornjak](#step-3-configuring-access-to-tornjak)
-5. [Cleanup](#cleanup)
+- [Step 0: Requirements](#step-0-requirements)
+- [Step 1: Setup of Deployment files](#step-1-setup-deployment-files)
+- [Step 2: Deployment of SPIRE and co-located Tornjak](#step-2-deployment-of-spire-and-co-located-tornjak)
+- [Step 3: Configuring Access to Tornjak](#step-3-configuring-access-to-tornjak)
+- [Cleanup](#cleanup)
 
 ## Step 0: Requirements
 
 We tested this quickstart with the following requirements:
  - [Minikube](https://minikube.sigs.k8s.io/docs/start/) version 1.12.0
  - [Docker](https://docs.docker.com/get-docker/) version 20.10.23
 
-## Step 1: Setup quickstart tutorial
+## Step 1: Setup deployment files
```

Comment:
We should stay consistent with title naming and the contents "Setup Deployment Files"